app/controllers/questions_controller.rb

```diff
@@ -65,7 +65,10 @@ def destroy
   end
 
   def bulk_destroy
-    @questions = load_selected_objects(Question)
+    @questions = Question.accessible_by(current_ability)
```

Comment:
We should be specifying the `:destroy` permission here, and perhaps testing that it is being applied; i.e. by throwing in an object that a user can :index but not :destroy and ensuring that doesn't get included. Questions that are published for instance.